Blackjack/Controller.cs

```diff
@@ -115,42 +102,33 @@ public void Play()
                     var dealerCard = DealCard(_dealer);
                     _output.WriteLine(String.Format(Messages.DealerCard, OutputFormatter.DisplayCard(dealerCard)));
                     dealerScore = Score.Calculate(_dealer.Hand);
-                    if (Rules.IsBlackjack(dealerScore))
-                        _output.WriteLine(String.Format(Messages.Dealer, Messages.Blackjack, OutputFormatter.DisplayHand(_dealer.Hand)));
-                    if (Rules.IsBust(dealerScore))
-                        _output.WriteLine(String.Format(Messages.Dealer, Messages.Bust, OutputFormatter.DisplayHand(_dealer.Hand)));
-                    else 
-                        _output.WriteLine(String.Format(Messages.Dealer, dealerScore, OutputFormatter.DisplayHand(_dealer.Hand)));
+                    DisplayDealerInformation(dealerScore);
                 }
             }
             while (!ShouldTurnEnd(choice, dealerScore));
```

Comment:
move this up a line to avoid confusion - same for other `do while` loops https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/statements/iteration-statements#the-do-statement